src/styles/app-main/app-main.html

```diff
@@ -0,0 +1 @@
+<main class="app-main"></main>
```

Comment:
To check that it works correctly, you could use some elements within this component. We can also discuss this at some point in the course.